code/modules/paperwork/ticketmachine.dm

```diff
@@ -98,7 +114,8 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/ticket_machine, 32)
 /obj/item/assembly/control/ticket_machine
 	name = "ticket machine controller"
 	desc = "A remote controller for the HoP's ticket machine."
-	var/datum/weakref/linked //To whom are we linked?
+	///To whom are we linked?
+	var/datum/weakref/linked
```

Comment:
weakref vars should preferably end in ``_ref`` so it's easier to know they're a weakref, please do so if you have the time.